group/variables.tf

```diff
@@ -154,6 +154,12 @@ variable "placement_tenancy" {
   default     = "default"
 }
 
+variable "root_vol_encrypted" {
+  type        = bool
+  description = "Whether the root volume should be encrypted or not."
+  default     = false
```

Comment:
By default the [aws terraform provider](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_configuration#block-devices) will not encrypt root block devices in an `aws_launch_configuration` resource. Here we set the default value as `false` in order to replicate the default behavior in the aws terraform provider.